modules/gdscript/gdscript.cpp

```diff
@@ -787,11 +787,11 @@ Error GDScript::reload(bool p_keep_state) {
 	err = compiler.compile(&parser, this, p_keep_state);
 
 	if (err) {
+		_err_print_error("GDScript::reload", path.is_empty() ? "built-in" : (const char *)path.utf8().get_data(), compiler.get_error_line(), ("Compile Error: " + compiler.get_error()).utf8().get_data(), false, ERR_HANDLER_SCRIPT);
```

Comment:
Right now we don't get compilation-related warnings in the editor. This line enables them. This does not mean analyzer errors, which presumably there are none of.

Compilation errors, if the analyzer passes, are _our_ problem, not a game developer's, so I understand the desire to hide them. But if the error only occurs when game devs run the game, that just puts a barrier towards bug reporting/fixing.